src/subprocess_launcher.rs

```diff
@@ -0,0 +1,65 @@
+#[cfg(any(feature = "v2_40", feature = "dox"))]
+use SubprocessLauncher;
+#[cfg(any(all(feature = "v2_40", not(windows)), feature = "dox"))]
+use ffi;
+#[cfg(any(feature = "v2_40", feature = "dox"))]
+use glib;
+#[cfg(any(feature = "v2_40", feature = "dox"))]
+use glib::object::IsA;
+#[cfg(any(all(feature = "v2_40", not(windows)), all(feature = "dox", unix)))]
+use std::os::unix::io::IntoRawFd;
+#[cfg(all(feature = "dox", not(unix)))]
+/// Replacement for "real" [`IntoRawFd`] trait for non-unix targets.
+///
+/// [`IntoRawFd`]: https://doc.rust-lang.org/std/os/unix/io/trait.IntoRawFd.html
+pub trait IntoRawFd {}
```

Comment:
Your check don't allow doc comments here, you need add it to lgpl-docs, but not sure that it restored in right place.
Also IMHO better not mixing `windows` and `unix` and use only `unix` (in all this file)